CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.3] - 2021-07-02
```

Comment:
Would it make sense to bump to 0.2.0 due to _Removed_ stuff?

Reply:
Yes good point. I'll recreate this PR.